Doc/library/functools.rst

```diff
@@ -338,26 +346,66 @@ The :mod:`functools` module defines the following functions:
    Roughly equivalent to::
 
       def partial(func, /, *args, **keywords):
-          def newfunc(*fargs, **fkeywords):
-              newkeywords = {**keywords, **fkeywords}
-              return func(*args, *fargs, **newkeywords)
+          def newfunc(*more_args, **more_keywords):
+              return func(*args, *more_args, **keywords, **more_keywords)
```

Comment:
This is wrong.

The difference is that `{**keywords, **fkeywords}` allows duplicates, but in function call this is an error.